fdbserver/DDTeamCollection.actor.cpp

```diff
@@ -5655,6 +5658,82 @@ class DDTeamCollectionUnitTest {
 		return Void();
 	}
 
+	ACTOR static Future<Void> GetTeam_TrueBestLeastReadBandwidth() {
+		Reference<IReplicationPolicy> policy = Reference<IReplicationPolicy>(
+		    new PolicyAcross(1, "zoneid", Reference<IReplicationPolicy>(new PolicyOne())));
```

Comment:
I think it's not allowed because `/mnt/ephemeral/xwang/readawareFdb/flow/FastRef.h:94:7: error: member 'delref' found in multiple base classes of different types`.
We define PolicyAcross as 
`struct PolicyAcross final : IReplicationPolicy, public ReferenceCounted<PolicyAcross>` rather than `struct PolicyAcross final : IReplicationPolicy` is strange I think, giving that `struct IReplicationPolicy : public ReferenceCounted<IReplicationPolicy>` is already reference counted...